iac-extensions/docker/src/test/resources/checks/SecretsHandlingCheck/SecretsHandlingCheck_arg.dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM foo
+
+
+# Noncompliant@+1 {{Make sure that using ARG to handle a secret is safe here.}}
+ARG ACCESS_TOKEN=ThisIsSomethingThatShouldProbablyBeSecret
+#   ^^^^^^^^^^^^
```

Comment:
Possible missing usecase in lowercase : `ARG access_token=ThisIsSomethingThatShouldProbablyBeSecret`

Reply:
Will add.